internal/dbutil/nulltime.go

```diff
@@ -48,3 +49,18 @@ func (nt NullTime) Value() (driver.Value, error) {
 	}
 	return nt.Time, nil
 }
+
+// parseSqliteTimeString parses sqlite times string.
+// It tries to process value as string with timezone first,
+// then fallback to parsing as string without timezone.
```

Comment:
it looks like it's falling back to parsing as a date instead of as a timestamp without timezone- maybe the comment can be updated? (but maybe just make a tiny pr to fix or something since this has been open for a while)